docs/testing.md

```diff
@@ -1,3 +1,34 @@
+# Automated Tests
+
+These tests run automatically once you have set up a working instance of Emblem.
+
+## Content API
+
+Content API unit tests run automatically under two circumstances:
+* [nightly against `main`](/terraform/ops/triggers/cloud_build_triggers.tf#:~:text=api_unit_tests_build_trigger)
+* [on pull requests to `main`](/terraform/ops/triggers/cloud_build_triggers.tf#:~:text=api_push_to_main_build_trigger)
+
+### Pipelines / Delivery System
+  
+The **delivery system** is covered by an automatic **End-to-End (E2E) Test**.
+
+This test uses Cloud Build to run the [setup/deployment script](/setup.sh), and checks that it completes without any errors.
+  
+These tests run against the `main` branch on a **nightly** basis.
+
+> **Note:** this is currently [in progress](https://github.com/GoogleCloudPlatform/emblem/issues/347)
```

Comment:
If we're going to merge this in the docs, let's remove future things or hold this in waiting until it's true.

Reply:
👍 let's do the latter

Reply:
I linked to the (optional) `installation-testing` directory instead.

I'll include a nightly build trigger (via Terraform) in there, via a separate PR.